torch_harmonics/distributed/distributed_sht.py

```diff
@@ -164,7 +165,12 @@ def forward(self, x: torch.Tensor):
         x = torch.view_as_real(x)
 
         # contraction
-        xs = torch.einsum('...kmr,mlk->...lmr', x, self.weights.to(x.dtype)).contiguous()
+        out_shape = list(x.size())
```

Comment:
this only modifies the RealSHT. We should also modify InverseSHT and VectorSHTs accordingly

Reply:
Those already do the same for distributed and serial.

Reply:
This is resolved in inverse sht as well